tests/contracts/hello/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hello"
+version = "0.0.0"
+authors = ["Oasis Protocol Foundation <info@oasisprotocol.org>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[lib]
+crate-type = ["cdylib"]
```

Comment:
any plans to support WASI so that I'm not limited to the limited selection of no_std crates?

Reply:
Yeah the thing is built with the idea that there can be additional ABIs in the future.